tensorrt_llm/_torch/visual_gen/attention_backend/parallel.py

```
@@ -699,9 +700,54 @@ def __init__(
+            try:
+                from tensorrt_llm.bindings.internal.userbuffers import (  # type: ignore
+                    ub_allocate, ub_is_initialized, ub_supported,
+                    userbuffers_send, userbuffers_recv,
+                )
+                if ub_supported() and ub_is_initialized():
+                    self._ub_p2p_ready = True
+                    self._ub_allocate = ub_allocate
+                    self._ub_send_fn = userbuffers_send
+                    self._ub_recv_fn = userbuffers_recv
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)